src/common/utils.js

```diff
@@ -100,10 +103,32 @@ export function copyToClipboard(data) {
   textField.remove();
 };
 
+/* Returns date set to local timezone and in format of locale */
+export function formatLocalDate(date, isUTC, timezone) {
+  if(!timezone) {
+    timezone = moment.tz.guess();
+  }
+  if (isUTC) {
+    date = moment.utc(date);
+  }
+  var localDate = moment.tz(date, timezone);
+  return localDate.format("l") + " " + localDate.format("LTS");
+}
+
+/* Returns date set to local timezone and formatted in standard format of YYYY-MM-DDTHH:mm:ss */
+export function convertToLocalTime(date, timezone) {
+  if(!timezone) {
+    timezone = moment.tz.guess();
+  }
+  return moment.tz(date, timezone).format("YYYY-MM-DDTHH:mm:ss");
+}
+
 export default {
   isFunction,
   debounce,
   formatDate,
   formatString,
-  getRandomString
+  getRandomString,
+  formatLocalDate,
+  convertToLocalTime
```

Comment:
formatLocalDate and convertToLocalTime show both date and time. In css classname we use ```className="last-refreshed-time"``` It would be great if the method names are consistent either *LocalDate or *LocalTime or *LocalDateTime